public/js/index.js

```diff
@@ -9,6 +9,21 @@ const getNextId = function () {
   return intId + 1
 }
 
+const positionWidgets = function () {
```

Comment:
It would be very helpful if each function, especially each non arrow function, had a brief description of its purpose and parameters. I could give much better feedback if I knew more about what it's supposed to do. Nudge, nudge.

Reply:
done. I'll add more as I go. In this case. no params